homeassistant/components/awair/strings.json

```diff
@@ -18,11 +18,11 @@
     },
     "error": {
       "auth": "[%key:common::config_flow::error::invalid_access_token%]",
```

Comment:
Would be nice to rename the key to `invalid_access_token` and update in config_flow and tests

Reply:
done